tesseract_ros/tesseract_examples/tesseract_ros_examples/launch/car_seat_demo.launch

```diff
@@ -19,5 +19,9 @@
   <node pkg="tesseract_ros_examples" type="tesseract_ros_examples_car_seat_demo" name="tesseract_ros_examples_car_seat_demo">
     <param name="plotting" type="bool" value="$(arg plotting)"/>
   </node>
+
+  <!-- Launch visualization -->
+  <node if="$(arg plotting)" pkg="rviz" type="rviz" name="tesseract_ros_examples_car_seat_demo_rviz"
```

Comment:
I think I would launch rviz either way. Or maybe add a viz flag or something. Then you could see the results in rviz but not have to step through the optimization by hitting enter each time. 

Previously this flag just set whether or not the plotting callback was added.

Reply:
I will add them back.